package.json

```diff
@@ -21,7 +21,7 @@
   },
   "dependencies": {
     "coffeescript": "github:helixbass/coffeescript#63d127c",
-    "prettier": "github:helixbass/prettier#72ae1f97"
+    "prettier": "github:helixbass/prettier#9105f5f1"
```

Comment:
Updated `prettier` dependency to include passing `options` to `willPrintOwnComments()` (opened https://github.com/prettier/prettier/pull/6081)